src/services/dashboard.js

```diff
@@ -258,50 +287,55 @@ export async function queryIterationSamples(params) {
 export async function queryTimeseriesData(payload) {
   const { selectedDateRange, selectedIterations } = payload;
 
-  const endpoint = `${endpoints.elasticsearch}/${getAllMonthsWithinRange(
+  const indices = `${parseMonths(
```

Comment:
Why the change from `getAllMonthsWithinRange` to `parseMonths`?

Reply:
> Why the change from `getAllMonthsWithinRange` to `parseMonths`?

Ah; and that's your unit test failure, because there _is_ no `parseMonths`.

Reply:
This is interesting I dont seem to recall how this change came while rebasing

Reply:
@npalaska ?